tensornetwork/network_components.py

```diff
@@ -1743,6 +1776,38 @@ def disconnect(edge,
   return edge.disconnect(edge1_name, edge2_name)
 
 
+def _merge_axis_names(
+    node1: BaseNode, 
+    node2: BaseNode, 
+    axes1: List[Text], 
+    axes2: List[Text]) -> None:
```

Comment:
I think these should be `int`s based on how you use them below.

Reply:
ah right, you are correct